source/Sashimi.Tests/BundledCliFixture.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Tests.Shared.Helpers;
+using FluentAssertions;
+using NUnit.Framework;
+using Sashimi.Tests.Shared;
+
+namespace Sashimi.Terraform.Tests
+{
+    [TestFixture]
+    public class BundledCliFixture
+    {
+        internal const string TerraformVersion = "0.11.8";
+
+        //Note that the CLI package may not end up in the test build folder when running locally.
+        [Test]
+        [WindowsTest]
```

Comment:
Only runs in [Windows](https://build.octopushq.com/project.html?projectId=OctopusDeploy_LIbraries_Sashimi_SashimiTerraform_CalamariNetFxWindowsTesting&testNameId=4237471779368549967&tab=testDetails)